src/sicmutils/mechanics/hamilton.cljc

```diff
@@ -83,15 +83,17 @@
             v (/ (- w b) M)]
         (- (* w v) (F v))))))
 
-(def Legendre-transform (make-operator Legendre-transform-fn "Legendre-transform"))
```

Comment:
A little catch I saw when investigating the `env` namespace! The operator name should be a symbol, not a string.